internal/server/task.go

```diff
@@ -70,6 +72,7 @@ func Task(client TaskClient, tmpl template.Template) Handler {
 				return err
 			}
 			data.Entity = caseitem.Summary()
+			data.Case = caseitem
```

Comment:
`data` is used to pass things to the template, so I think we should avoid adding things to it that aren't used there.

Instead, you could create `var caseitem sirius.Case` above `group.Go`, replace the `:= client.Case` with `= client.Case`, and then use `caseitem` rather than `data.Case` when checking the type below.